### DISCUSSION TITLE
DMUCS GUI Not writing out separated files

### DISCUSSION THREAD

**Discussion by tcrnottingham** [CarlGao4/Demucs-Gui#34] [Q&A]
Hi,
I've downloaded the CPU version 0.1 and installed OK. When it starts it says ffmpeg is not available (but it is on my drive C in the ffmpeg folder). I gave it my first .mp3 to separate and it appears to progress from 1/5 through 5/5 but puts a message in the bottom left saying "Failed to write with soundfile, using wave instead".
I can't find any .wav files, so where does it put them? (or has it failed).

Thanks

TcrNottingham

**Comment by CarlGao4** [comment]
Sorry, but please use 1.0a1 instead.

**Reply by tcrnottingham** [reply]
1.0a1 is quarantined by AVG antivirus as infected with Win64:Evo-gen[Trj].
Has anyone else seen this?

**Reply by CarlGao4** [reply]
See https://github.com/CarlGao4/Demucs-Gui/blob/main/usage.md#the-application-triggers-my-antivirus-software
